testing/e2e/workload_test.go

```diff
@@ -660,7 +660,10 @@ func TestCreateFromGitWithAnnotations(t *testing.T) {
 			Name:         "Update the created workload",
```

Comment:
Can we add a test case to create workload with just --git commit and/ or with combination ?

Reply:
It ust have got deleted when I rebased this PR. Thanks for pointing it Shaheer.